index.js

```diff
@@ -268,7 +269,7 @@ const TeacherIDLookup = () => {
           }}
           className="active"
         >
-          <strong>1</strong> <small>Find Your School</small>
+          <strong>1</strong> <small>Find Your School - Enter the keyword of your school's name example - Kennedy for John F Kennedy</small>
```

Comment:
I don't think that's where we want that to go. See this screenshot for how it looks: 
https://www.dropbox.com/s/8qn6ccvxqef08qs/Screenshot%202020-04-14%2020.15.23.png?dl=0